test/src/test/java/org/corfudb/infrastructure/ManagementServerTest.java

```diff
@@ -72,21 +72,4 @@ public void bootstrapManagementServer() {
         assertThat(getLastMessage().getMsgType()).isEqualTo(CorfuMsgType.MANAGEMENT_ALREADY_BOOTSTRAP_ERROR);
     }
 
-    /**
-     * Triggering the failure handler with and without bootstrapping the server.
-     */
-    @Test
-    public void triggerFailureHandler() {
```

Comment:
FYI this test is broken to begin with because the AbstractServerTest infrastructure doesn't block. I removed it...